### ISSUE TITLE
Shutter angle in G0Pro 13 not working right

### ISSUE THREAD

**Issue by DTL77** [gopro/labs#1196]
This concerns the use of EXPT. It does not work correctly for GoPro 13 at 24fps and all at 360.

Angle  Target    GP13-24fps  GP10/11-24 fps   Target            GP13-30 fps     GP10/11-30 fps
360      1/24      1/60              1/49                    1/30                1/61                 1/61
180      1/48      1/60              1/49                    1/60                1/61                 1/61
90        1/96      1/120            1/97                    1/120              1/121               1/122
45        1/192    1/194            1/194                   1/240             1/242               1/243

**Comment by dnewman-gpsw** [comment]
Sorry, I don't follow you notation.  What are the different columns?

**Comment by DTL77** [comment]
> Sorry, I don't follow you notation. What are the different columns?

Sorry the spacing got messed up. I had them in proper columns. So the first vertical set of numbers are the angles, the second what that represents for 24fps then values for GP13 and GP10/11 then target for 30fps and values for GP13 and then GP10/11.  So for example at 90 the value for 24 should be 1/96 but GP13 is 1/194 while the GP10/11 are close at 1/97. This help?

**Comment by dnewman-gpsw** [comment]
I think there confusion over what EXPT does. It doesn't set a fixed shutter angle, it sets a maximum shutter angle so it will still depend on light. Are you covering the lens to try and simulate what the maximum could be?

**Comment by DTL77** [comment]
Yes that is exactly what I did and rechecked many times before sending in the data.


David T. Long
Professor Emeritus & Adjunct Professor
Aqueous & Environmental Geochemistry
Dept. of Earth and Environmental Sciences
288 Farm Lane, Rm 207
Michigan State University
East Lansing, Michigan 48824
my office: 517.353.9618
dept office: 517.355.4626
email: ***@***.***

[cid:a70e7d48-716f-44df-a520-e02ed96d11b5]

Michigan State University occupies the ancestral, traditional, and contemporary Lands of the
      Anishinaabeg–Three Fires Confederacy of Ojibwe, Odawa, and Potawatomi peoples.
                   The University resides on Land ceded in the 1819 Treaty of Saginaw.

    Sometimes you win, sometimes you lose, sometimes it rains. (Bull Durham, 1988)
Spartan Strong
________________________________

**Comment by dnewman-gpsw** [comment]
This is a very old feature, before the cameras had GYRO compensated hypersmooth. It was added to assist users reduce the amount of motion blur in hypersmooth low light situations.  The gyro comp does that automatically now.  You can disable that with $GCMP=1.

**Comment by DTL77** [comment]
Interesting are you saying that GP13 will stabilize in low light. I have not noticed but will give it a try. Probably the same issue but one of the reasons I was using this was the situation in which in low light conditions at too low of a shutter speed, points of light flicker as one walked (e.g., POV at night in a city). I solved that by turning off hype smooth and setting a speed limit so that would not happen. Then I used Player to stabilize. However, for some reason Player lowers the bit rate (I set to 190, Player knocks it down to < 50) so started using GyroFlow. Bummer because I prefer Player.

Am I off track?


David T. Long
Professor Emeritus & Adjunct Professor
Aqueous & Environmental Geochemistry
Dept. of Earth and Environmental Sciences
288 Farm Lane, Rm 207
Michigan State University
East Lansing, Michigan 48824
my office: 517.353.9618
dept office: 517.355.4626
email: ***@***.***

[cid:d70bd4ab-a1de-480f-acfc-6f3b39686cb2]

Michigan State University occupies the ancestral, traditional, and contemporary Lands of the
      Anishinaabeg–Three Fires Confederacy of Ojibwe, Odawa, and Potawatomi peoples.
                   The University resides on Land ceded in the 1819 Treaty of Saginaw.

    Sometimes you win, sometimes you lose, sometimes it rains. (Bull Durham, 1988)
Spartan Strong
________________________________

**Comment by DTL77** [comment]
Great )

**Comment by dnewman-gpsw** [comment]
Yes it "sort of" a bug in HERO13. HERO11 and 12 only seems to go to 180°, no 360° any more (really bad for HS.) It is likely by design on HERO13, although I don't like it either. Seems they are using 30 or 25fps sensor timing for 24 modes. This means that only have to tune the camera for the PAL/NTSC, not independently for 24fps modes. If you scan p25!1Np24, this will switch the camera to PAL timing, and now 24p uses 1/50 (likely proving my tuning theory.) Labs is still doing the correct thing, but is a lower level that is flawed. I will file a report internally.

**Comment by DTL77** [comment]
Thanks David! Always something 🙂.

dave

David T. Long
Professor Emeritus & Adjunct Professor
Aqueous & Environmental Geochemistry
Dept. of Earth and Environmental Sciences
288 Farm Lane, Rm 207
Michigan State University
East Lansing, Michigan 48824
my office: 517.353.9618
dept office: 517.355.4626
email: ***@***.***

[cid:ea1a3724-3230-4f47-a4f9-8af58440554d]

Michigan State University occupies the ancestral, traditional, and contemporary Lands of the
      Anishinaabeg–Three Fires Confederacy of Ojibwe, Odawa, and Potawatomi peoples.
                   The University resides on Land ceded in the 1819 Treaty of Saginaw.

    Sometimes you win, sometimes you lose, sometimes it rains. (Bull Durham, 1988)
Spartan Strong
________________________________

**Comment by DTL77** [comment]
PS in terms of my testing the 11 worked fine, except for the 360.

**Comment by DTL77** [comment]
sorry I meant the 10 for my most recent tests

**Comment by DTL77** [comment]
Hi David, sorry to bother. I guess gyro stabilization is essentially HyperSmooth from what I learned. In my experience (GP 11, 13; also have the 9 & 10, did not get the 12) stabilization did not work well in low light. I did not want to use a gimbal for POV videos (e.g., city streets) and learned about the influence of shutter speed in stabilization. Thus, setting the shutter angle via Labs came into play (I have been a Labs user almost since my first GoPro, it is great and Labs sets GoPro, INMO, above the rest). I had just done a trip to Chicago with the 13 and had been using the 45 degree angle, which is recommended. Works ok but of course it is still a bit dark (I also turn Noise Reduction of via Labs). Back home I thought why not try 90 degrees and that is when I recognized the issue,  instead of 1/96 it was 1/120. 180 was off also which is too bad for the 180 rule. Now this is for 24 as 30 is fine.  Anyway I though I would explain where I was coming from. I am going to try what you outlined. If you are correct, from what I see I think you are correct, it is strange that GP is using 30 or 25fps sensor timing for 24 modes. I think because 45 is recommended that others who use this approach  would not notice the issue.  Hope this can get resolved. Somehow I don't see it happening tomorrow, since it is not a Lab issue. Thanks for your insights, Dave. PS if you ever need a Beta Tester, count me in :).
